src/store/actions/index.js

```diff
@@ -68,39 +69,28 @@ export const getVideos = (token) => (dispatch) => {
     });
 };
 
+export const POST_VIDEO_START = "POST_VIDEO_START";
+export const POST_VIDEO_SUCCESS = "POST_VIDEO_SUCCESS";
+export const POST_VIDEO_ERROR = "POST_VIDEO_ERROR";
+
+export const postVideo = (token, video, history) => (dispatch) => {
+  dispatch({ type: POST_VIDEO_START });
+  axiosWithAuth(token)
+    .post(`${urlServer}/videos`, video)
+    .then((res) => {
+      setTimeout(() => {
+        dispatch({ type: POST_VIDEO_SUCCESS });
+        history.push(`/video/${res.video.id}`);
+      }, 1000);
+    })
+    .catch((err) => dispatch({ type: POST_VIDEO_ERROR,  err }));
+};
+
 export const LOGOUT = "LOGOUT";
```

Comment:
This looks like what we want.  Pushing to the route you have on line 83 will direct the view to the SinglVideoPage. 

A couple things to roll around:
 - I have the route for a singlevideo defined at /video(s)/videoID. Need to update that before pushing up the SingleVideoPAge PR.
 - I didn't catch there was a REACT_APP_SERVER_URL already when I created BASE_URL in axioswithauth.
-- Glad to see we're all on the same page with the security. 
-- will refactor to use SERVER_URL in both instances.
 - How do units tests play with actions files? or do they?
 Maybe somebody can get a unit test in for CreateVideo view.
--- I can help with this and I could see putting it off until after we get functionality for RC1 completed
--- assert that we put out the correct data shape to send with the createVideo POST
--- assert the video id clicked on matches the video ID in the body of the POST req. 
--- use mocks and stubs for the database, backend API, actual server calls etc.